src/coreclr/nativeaot/Runtime/MathHelpers.cpp

```diff
@@ -56,29 +56,33 @@ FCIMPL1_D(uint32_t, RhpDbl2UInt, double val)
 FCIMPLEND
 
 #ifndef HOST_64BIT
-EXTERN_C int64_t QCALLTYPE RhpLDiv(int64_t i, int64_t j)
+FCIMPL2(int64_t, DivInt64Internal, int64_t i, int64_t j)
```

Comment:
Does this need `_VV` flavor of the macro to get the arguments ordered correctly on all arches?

Reply:
Agh, I thought I had done that.